scripts/checker.sh

```diff
@@ -3,5 +3,20 @@ set -e
 set -u
 set -o pipefail
 
-# Do the lifeness check exit status 0 -> success | 1 -> fail
-searchd --status 1> /dev/null && exit 0 || exit 1
\ No newline at end of file
+# mountpoint with readiness probe file
+MOUNT=$(realpath "${PROBE_MOUNTPOINT:-/tmp}")
+READY_FILE="${MOUNT}/checker_ready.txt"
+
+clean_probe_files() {
+        rm -f "${READY_FILE}" || :
+}
+
+# source this stuff until here
+[ "$0" = "${BASH_SOURCE[*]}" ] || return 0
+
+if searchd --status 1> /dev/null; then
+    echo "READY" > "${READY_FILE}"
+    exit 0
+else
+    exit 1
```

Comment:
Here we should remove the ready file to mark the service as un ready when unhealthy

Reply:
good point, yes!